hawkular-metrics/hawkular-metrics-wrapper.sh

```diff
@@ -147,4 +149,5 @@ exec 2>&1 /opt/jboss/wildfly/bin/standalone.sh \
   -b `hostname -i` \
   -bprivate `hostname -i` \
   -P $HAWKULAR_METRICS_DIRECTORY/server.properties \
+  --properties=/configuration/hawkular-metrics.properties \
```

Comment:
Is Wildfly able to deal with multiple `--properties` parameters? Also, it would be nice to either change the previous like to be `--properties` as well, or this to be `-P`, for consistency's sake.